### PR TITLE
fix: move board styles out of variants

### LINKED ISSUE #146: Variants of some UI Kit boards are disabled in Add Panel
Variants from buttons and components boards are disabled for usage in Add Panel.

Screenshot 2024-10-14 at 14 03 00

The reason is that those variants use board styles:

Screenshot 2024-10-14 at 14 02 06

### PR DESCRIPTION
**Pull request by tetianaman**
Fixes #146

### CODE REVIEW

**Comment by Letushev** [comment]
> Fixes https://github.com/codux-templates/e-commerce-reclaim/pull/143

It fixes another PR? Which one do we need or both?

**Comment by tetianaman** [comment]
> > Fixes #143
> 
> It fixes another PR? Which one do we need or both?

omg, I thought it was an issue ticket which Sagiv created for me. Let me recreate a ticket

File: _codux/ui-kit/buttons/buttons.board.module.scss [hunk 1]
```diff
@@ -25,32 +25,3 @@
         width: 100%;
     }
 }
-
-.trashIcon {
-    height: 27px;
-    padding-top: 2px;
```
Comment (Letushev) [inline _codux/ui-kit/buttons/buttons.board.module.scss:31]: You had paddings here and there. Does it look fine without them?

Comment (tetianaman) [reply _codux/ui-kit/buttons/buttons.board.module.scss:31]: I think they are fine. Look
With paddings:
Screenshot 2024-10-14 at 17 59 18
No paddings:
Screenshot 2024-10-14 at 17 59 44

Comment (tetianaman) [reply _codux/ui-kit/buttons/buttons.board.module.scss:31]: Ok, I've moved Menu icon 4px down using padding. Now is better:
Screenshot 2024-10-14 at 18 20 01

**Review by Letushev — APPROVED** [review]

**Pull request merged**